### ISSUE TITLE
Achievement "A Lingering Fragrance" tracking

### ISSUE THREAD

**Issue by codemasher** [MadeBaruna/paimon-moe#350]
Hi, i got a commission related to the above achievement today for the first time and it seems there a series of prerequisites required:

First, there's 3 versions of the daily commission [Desert Bloom](https://genshin-impact.fandom.com/wiki/Desert_Bloom):
- [ ] Watering Version
- [ ] Kitten Version
- [ ] Zaytun Peach Version

After that, the commission [Blooming Sands: Lasting Scent](https://genshin-impact.fandom.com/wiki/Blooming_Sands:_Lasting_Scent), which grants the achievement, will be unlocked.

It would be nice if all of this would be trackable!

**Comment by MadeBaruna** [comment]
Hello, did you get the achievement? (A Lingering Fragrance)

**Comment by codemasher** [comment]
No, not yet. I'm going by several comments re the prerequisites, e.g. [this one](https://genshin-impact.fandom.com/wiki/Desert_Bloom?commentId=4400000000000385334).

**Comment by codemasher** [comment]
I got the "Lasting Scent" commission and the related achievement today. I did all 3 of the mentioned prerequisite quests before.

![image](https://user-images.githubusercontent.com/592497/212078252-fc535ed1-5387-43d3-8bae-2b29dbd45c6b.png)
